src/compiler/diagnosticMessages.json

```diff
@@ -3603,6 +3603,18 @@
         "code": 6199,
         "reportsUnnecessary": true
     },
+    "Definitions of the following identifiers conflict with those in another file: {0}": {
+        "category": "Error",
+        "code": 6200
+    },
+    "Conflicts are in this file.": {
+        "category": "Message",
+        "code": 6201
+    },
+    "Conflicts here.": {

```

Comment:
@DanielRosenwasser comments on the messages?

Reply:
this is a bit whimsical, but would be nice if it looked like:

```txt
file1.ts (1, 2): Error: Duplicate identifier 'foo'.
  + file2.ts (2, 3): "foo" was also declared here.
  + file3.ts (2, 3): and here.
```

Reply:
In what output? The error baselines?

Reply:
Ohh, i get it, the different messages. Certainly doable.

Reply:
Done/10 with 🍚